src/ui/EditTask.scss

```diff
@@ -141,15 +135,15 @@
     justify-content: space-between;
 }
 
-.tasks-modal {
-    .tasks-modal-error {
-        border: 1px solid red !important;
-    }
-
-    .tasks-modal-warning {
-        color: var(--text-warning) !important;
-        background-color: rgba(var(--background-modifier-warning-rgb), 0.2) !important;
-    }
+.tasks-modal-buttons {
+    position: sticky;
+    bottom: 0;
+    background-color: var(--modal-background);
+    padding-bottom: 16px;
+    padding-top: 16px;
+    display: grid;
+    grid-template-columns: 3fr 1fr;
+    column-gap: .5em;
```

Comment:
@claremacrae please note that there is no change of behaviour here - the `tasks-modal` part went up to the beginning of the file, and `tasks-modal-button` was moved down to the bottom of the file

Reply:
Got it.